README.md

```diff
@@ -46,6 +48,29 @@ module.exports = {
 }
 ```
 
+If you need to use options with `css-loader`.
+
+```js
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: ExtractTextPlugin.extract({
+          fallback: "style-loader",
+          use: 'css-loader?modules,localIdentName="[name]-[local]-[hash:base64:6]"'
```

Comment:
@pragyandas Are you on `extract-text-webpack-plugin > = v2.0.0-rc.3` ? The options in `{Object}` notation looks nearly the same as the one I use everyday, the only diverence is that they are in an `{Array}`, bc I have other loaders in my setup. Is `use` allowed to take an `{Object}` as value?

```diff
+ use: [
  {
     loader: 'css-loader',
     options: {
       modules: true,
       localIdentName: '[name]-[local]-[hash:base64:6]'
    }
  }
+ ]
```